e2e_test/udf/js_udf.slt

```diff
@@ -0,0 +1,65 @@
+statement ok
+create function int_42() returns int language javascript as $$
+    return 42;
+$$;
+
+statement ok
+create function gcd(a int, b int) returns int language javascript as $$
+    if(a == null || b == null) {
+        return null;
+    }
```

Comment:
note: we will support `RETURNS NULL ON NULL INPUT` keywords soon. With this feature, the if branch can be omitted and the overall performance can be improved.

Reply:
> we will support RETURNS NULL ON NULL INPUT keywords soon

It's ambiguous to me: Does it return null when **all** arguments are null, or when **any** arguments are null?

Reply:
any
> RETURNS NULL ON NULL INPUT or STRICT indicates that the function always returns null whenever **any** of its arguments are null. If this parameter is specified, the function is not executed when there are null arguments; instead a null result is assumed automatically.